java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPAddress.java

```diff
@@ -169,4 +183,26 @@ else if (address instanceof Inet6Address)
     public static IPAddress<?> fromInetAddress(@Nonnull InetAddress address) {
         return of(address);
     }
+
+    @Nonnull
+    protected <T extends InetAddress> T toInetAddressSubclassChecked(
```

Comment:
I was thinking if `getBytes()` misbehaves we'll have the debug info here. But you're right, they should be validated through unit tests and it's not worth the complexity here. I'll make the change. Thanks!